configs/rpt/simpleusb.conf

```diff
@@ -116,6 +116,11 @@ clipledgpio = 1                     ; Enable ADC Clip Detect feature to use a GP
                                     ; Supports URI Clip LED by setting a GPIO (if available) high for 500mS
                                     ; when clipping detected. Value = GPIO# to use (GPIO1 recommended)
 
+legacyaudioscaling = no             ; If yes, continue to do raw audio sample scaling and clipping, resulting in Tx audio levels increasing
```

Comment:
Add here that the default is currently yes but will eventually be no and/or setting removed